conf/messages.en

```diff
@@ -1035,6 +1035,11 @@ totalValueQuotedShares.error.required = Enter the total value of all quoted shar
 totalValueQuotedShares.error.invalid = Total value of all quoted shares must be an amount of money, like 123.45 or 156
 totalValueQuotedShares.error.tooLarge = Total value of all quoted shares must be 999999999.99 or less
 
+totalValueQuotedSharesCYA.title = Check your answers
```

Comment:
We have a generic `checkYourAnswers.title` and `checkYourAnswers.heading` for these two